addons/web/static/src/views/kanban/progress_bar_hook.js

```diff
@@ -161,16 +160,15 @@ class ProgressBarState {
             const currencies = group.aggregates?.[aggregateField.currency_field];
             if (currencies?.length > 1) {
                 return {
-                    title: `${title}: ${_t("different currencies cannot be aggregated")}`,
                     value,
-                    currency: false,
+                    currencies,
                 };
             }
             if (currencies?.[0]) {
                 return {
                     title,
                     value,
-                    currency: getCurrency(currencies[0]),
+                    currencyId: currencies[0],
```

Comment:
uniformize API? Always give `currencies` which is the list of currency ids which may be the empty list => formatInteger, a list with one currency => formatCurrency and a list of more than one currency => multi currency popover